docs/development.md

```diff
@@ -146,14 +151,9 @@ make delete-workload-cluster
 
 ### Manual Testing
 
-#### Setting up the environment
-
-Your environment must have the Azure credentials as outlined in the [getting
-started prerequisites section](./getting-started.md#Prerequisites)
-
 #### Creating a dev cluster
 
-The steps below are provided in a convenient script in [hack/create-dev-cluster.sh](hack/create-dev-cluster.sh). Be sure to set `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`, `AZURE_SUBSCRIPTION_ID`, and `AZURE_TENANT_ID` before running. Optionally, you can override the different cluster configuration variables. For example, to override the workload cluster name:
+The steps below are provided in a convenient script in [${REPO_ROOT}/hack/create-dev-cluster.sh](../hack/create-dev-cluster.sh). Be sure to set `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`, `AZURE_SUBSCRIPTION_ID`, and `AZURE_TENANT_ID` before running. Optionally, you can override the different cluster configuration variables. For example, to override the workload cluster name:
```

Comment:
do we need ${REPO_ROOT} in the display name? I thought it was implied (also all our other links in docs don't have it)

Reply:
you're right, I was just looking at the bits about repo setup and didn't want to confuse people

Reply:
fixed + squashed